app/Jobs/RockTheVote/ImportRockTheVoteRecord.php

```diff
@@ -134,20 +130,40 @@ private function getUser($id, $email, $mobile)
     }
 
     /**
-     * Creates new user with record user data.
+     * Creates new user and post with record data.
      *
-     * @param array $data
-     * @return NorthstarUser
+     * @return void
      */
-    private function createUser()
+    private function handleNewUser()
```

Comment:
Why the change in method name? Prior name seemed to follow with `createPost` and using the typical CRUD verbs.

Reply:
Because this function creates a new user, but also a new post, a log record, and sends the password reset email if user subscribed. I thought it followed with along with `handle` function nicely, but could potentially rename to something like `createUserAndPost`

Reply:
So to me that highlights that maybe the location that calls this method should do the other things instead of mixing it all into one method. It's a better practice to keep methods like this doing a single thing.

Instead in the `handle()`method above, something like this?

```php

if (! $user) {

  $user = $this->createUser();

  $this->createPost($user);

  RockTheVoteLog::createFromRecord($this->record, $user, $this->importFileId);

  $this->sendUserPasswordResetIfSubscribed($user);
}
```

Reply:
I think that makes sense - will make this change.